catalyst_voices/packages/catalyst_voices_localization/lib/l10n/intl_en.arb

```diff
@@ -708,6 +708,20 @@
   "yes": "Yes",
   "no": "No",
   "total": "Total",
+  "file": "file",
+  "key": "key",
+  "upload": "Upload",
+  "browse": "browse",
+  "uploadDropInfo": "Drop your {itemNameToUpload} here or ",
```

Comment:
Please add @uploadDropInfo entry and specify the type of itemNameToUpload (should be string), otherwise it gets generated as object